dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -847,26 +848,49 @@ public String getResponseFileName() {
 
   @CheckForNull
   public String getDefinition(@Nonnull ConnectorProperty property) {
+    return getDefinitionMap().get(property.getName());
+  }
+
+  private Map<String, String> getDefinitionMap() {
     if (definitionMap == null) {
-      searchDefinitions();
+      definitionMap = new HashMap<>();
```

Comment:
Nit: It's better practice to build the Map into a local, and assign the local to the field _last_ because if building the Map throws some sort of exception, we get a dirty field. If we do it in a local, we get a dirty local, but we don't break the object state.

It's unlikely to matter in this case, but it's good to get into the habit overall.

Reply:
Fixed.